.github/workflows/publish.yml

```diff
@@ -68,10 +68,11 @@ jobs:
       - name: create PR to merge release branch and master
         run: |
           git fetch
-          git checkout release/${{ github.ref_name }}
-          git checkout -b merge_release/${{ github.ref_name }}
-          git push
-          gh pr create --fill
+          release_branch_version=$(git branch -a --contains ${{ github.ref_name }} | awk -F '/' '{print $NF}')
```

Comment:
when I execute this in local, it gives me twice the branch:
```
export release_branch_version=$(git branch -a --contains 'release/2.127.0' | awk -F '/' '{print $NF}')
❯ echo $release_branch_version
2.127.0
2.127.0
```

Reply:
I can't reproduce :/  I will try something else

Reply:
can you give me the result of: `git branch -a --contains 2.127.0`

thanks